Queue/Storage/Context.ts

```diff
@@ -2,45 +2,61 @@ import { gracely } from "gracely"
+	async enqueue(event: Types.EventBase): Promise<void> {
+		console.log("Context.enqueue", event)
+		const request: http.Request.Like = {
+			method: "POST",
+			url: event.url,
+			body: { hook: event.hook, event: event.body },
+			header: { ...event.header, contentType: "application/json" },
+		}
+		const maxRetries = event.options?.maxRetries ?? 5
+		const timeFactor = event.options?.timeFactor ?? 1
+		this.tryEnqueue(request, maxRetries, timeFactor)
 	}
-	private async tryEnqueue(request: http.Request.Like, retries = 0): Promise<void> {
-		await this.state.storage.put(`hook`, { request: request, retries: retries })
-		this.state.waitUntil(
-			this.state.storage.setAlarm(
-				isoly.DateTime.epoch(isoly.DateTime.nextSecond(isoly.DateTime.now(), this.timeFactor * retries), "milliseconds")
+	private async tryEnqueue(
+		request: http.Request.Like,
+		maxRetries: number,
+		timeFactor: number,
+		retries = 0
+	): Promise<void> {
+		if (gracely.Error.is(await this.send(request))) {
+			await this.state.storage.put(`hook`, { request, retries, maxRetries, timeFactor })
+			await this.alarm.set(
+				"dequeue",
+				isoly.DateTime.epoch(isoly.DateTime.nextSecond(isoly.DateTime.now(), timeFactor * retries), "milliseconds")
 			)
-		)
+		}
 	}
-	async dequeue(): Promise<http.Response | gracely.Error> {
-		let result: http.Response | gracely.Error
-		let data: { request: http.Request.Like; retries: number } | undefined
+	async dequeue(): Promise<void> {
+		let data: { request: http.Request.Like; maxRetries: number; timeFactor: number; retries: number } | undefined
 		if (
-			!(data = await this.state.storage.get<{ request: http.Request.Like; retries: number }>(`hook`)) ||
-			!data.request
-		)
-			result = gracely.server.databaseFailure("item not found")
-		else {
-			result = await this.send(data.request)
+			(data = await this.state.storage.get<{
+				request: http.Request.Like
+				retries: number
+				maxRetries: number
+				timeFactor: number
+			}>(`hook`)) &&
+			data.request
+		) {
+			const result = await this.send(data.request)
 			await this.state.storage.delete(`hook`)
-			if (gracely.Error.is(result) && data.retries < this.maxRetries)
-				this.tryEnqueue(data.request, ++data.retries)
+			if (gracely.Error.is(result) && data.retries < data.maxRetries)
+				this.tryEnqueue(data.request, data.maxRetries, data.timeFactor, ++data.retries)
 		}
-		return result
 	}
-	static open(state: cloudflare.DurableObjectState, maxRetries?: number, timeFactor?: number): Context {
-		return new Context(state, maxRetries, timeFactor)
+	static open(state: cloudflare.DurableObjectState): Context {
+		return new Context(state)
 	}
-	async send(request: http.Request.Like): Promise<http.Response | gracely.Error> {
+	async send(request: http.Request.Like): Promise<boolean> {
+		console.log("Context.send", request)
 		const response = await http.fetch(request)
-		return gracely.Error.is(response.body) ? response.body : response
+		return response.status == 200
```

Comment:
Why give false on all successes except for 200? I thing you want to do:
`response.status>=200 && response.status<300`